# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strongswan/fix-ar-usage.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

AR might be set to a cross toolchain -ar ...

  - Rene Rebe <rene@exactcode.de>

--- strongswan-2.4.4/lib/libcrypto/libblowfish/Makefile.vanilla	2005-06-22 20:33:19.000000000 +0200
+++ strongswan-2.4.4/lib/libcrypto/libblowfish/Makefile	2005-06-22 20:33:32.000000000 +0200
@@ -14,7 +14,7 @@
 MAKE=		make -f Makefile.ssl
 MAKEDEPEND=	$(TOP)/util/domd $(TOP)
 MAKEFILE=	Makefile.ssl
-AR=		ar r
+AR=		ar
 RANLIB=		ranlib
 PERL=		perl
 
@@ -58,7 +58,7 @@
 lib:	$(LIB)
 
 $(LIB):	$(LIBOBJ)
-	$(AR) $(LIB) $(LIBOBJ)
+	$(AR) r $(LIB) $(LIBOBJ)
 	$(RANLIB) $(LIB)
 
 # elf
